src/foam/core/JSON.js

```diff
@@ -640,101 +640,121 @@ foam.LIB({
   constants: {
 
     // Pretty Print
-    Pretty: foam.json.Outputter.create({
-      strict: false
-    }),
+    Pretty: function() {
```

Comment:
foam.json is a library, not a class/model, so there is no advantage to doing this (but it could me made to). Does this still work?

Reply:
I apply the require change in another PR.